inst/shiny/modules/tab_nca.R

```diff
@@ -149,14 +144,18 @@ tab_nca_server <- function(id, adnca_data, grouping_vars) {
             invokeRestart("muffleWarning")
           })
 
+          # Apply flag rules to mark results in the `exclude` column
+          flag_rules_to_apply <- purrr::keep(rules(), ~ .x$is.checked) |>
+            purrr::map(~ .x$threshold)
```

Comment:
nitpick: When using pipes, it is a bit nicer to start with the data itself - it is easier to see what is going into the pipe.
```suggestion
          flag_rules_to_apply <- rules()
            purrr::keep(~ .x$is.checked) |>
            purrr::map(~ .x$threshold)
```